.github/workflows/build.yml

```diff
@@ -12,19 +12,20 @@ on:
   workflow_dispatch:
     inputs:
       canary-release:
-        description: 'Canary release after build'
-        type: boolean
-        required: false
-        default: false
-      skip-checks:
-        description: 'Skip tests and checks'
+        description: 'Release canary version (skips tests and checks)'
         type: boolean
         required: false
+        # This looks like a boolean, but it's actually a string
```

Comment:
Isn't this comment now redundant/misleading when you've also switched to the inputs api and do actually check on booleans now?